db/models/ticker.go

```diff
@@ -7,7 +7,7 @@ type TickerQuery struct {
 
 type Ticker struct {
 	BasicTimeModel
-	Coin      uint   `gorm:"primary_key; auto_increment:false" sql:"index"`
```

Comment:
any reason to use string instead of uint?

Reply:
It was made to prevent any case when default value for coin will not affect functionality.

There were issues with generic bulk create function that doesn’t want to write default values without using pointers like *uint

I think it is shitcode, so I swapped it to manual bulk insert

Coin will be changed back to uint as well